routingpy/isochrone.py

```diff
@@ -64,9 +64,13 @@ def __init__(self, geometry=None, interval=None, center=None):
     @property
     def geometry(self):
         """
-        The geometry of the isochrone as [[lon1, lat1], [lon2, lat2], ...] list.
+        The geometry of the isochrone as [[[lon1, lat1], [lon2, lat2], ...]] list.
```

Comment:
isn't it `[[[[lon_ex1, lat_ex1], [lon_ex2, lat_ex2]], [[lon_in1, lat_in1], [lon_in2, lat_in2]], [other_poly]]`? i.e. multipoly, poly, ring (exterior or interior), coord as nested lists? That'd at least be the most future-proof way for isochrone responses.